packages/react-day-picker/src/contexts/Navigation/useNavigationState.ts

```diff
@@ -27,7 +28,6 @@ export function useNavigationState(): [
 
   // Update month if updated from context.
   React.useEffect(() => {
-    // TODO: return void also if `context.defaultMonth`?
```

Comment:
I removed this React.useEffect, because it's no longer necessary to use the internal state if it is controlled. Instead we just ignore the internal state

Reply:
Is there a reason that you added it back, or perhaps it was just a merge commit that added it back

Reply:
> Is there a reason that you added it back, or perhaps it was just a merge commit that added it back

Are we branching from the same master? I see that line added by me 8 months ago :)

But anyway you are right it can be safely removed - and if not, it is because we are missing the test :/

Reply:
Removed in 29043dc2dce92fe450b215e60271a5e532d43483.